assets/cubyz/shaders/block_entity/chest.vert

```diff
@@ -0,0 +1,113 @@
+#version 460
```

Comment:
These shaders look they are almost an exact copy of the chunk shaders.

This is unmaintainable (and also unreviewable). Please find a way to reuse the same shader, you can for example use preprocessor options to distinguish the things that are different.